readme.md

```diff
@@ -7,100 +7,185 @@
-| **[Code Samples](./examples)** |These simple examples demonstrate how to easily use the SDK to include code snippets that access the camera into your applications. Check some of the [**C++ examples**](./examples) including capture, pointcloud and more and basic [**C examples**](./examples/C) | Included in [**Intel.RealSense.SDK.exe**](https://github.com/IntelRealSense/librealsense/releases) |
-| **[Wrappers](https://github.com/IntelRealSense/librealsense/tree/development/wrappers)** | [Python](./wrappers/python), [C#/.NET](./wrappers/csharp) API, as well as integration with the following 3rd-party technologies: [ROS1](https://github.com/IntelRealSense/realsense-ros/tree/ros1-legacy), [ROS2](https://github.com/IntelRealSense/realsense-ros/tree/ros2-master), [LabVIEW](./wrappers/labview), [OpenCV](./wrappers/opencv), [PCL](./wrappers/pcl), [Unity](./wrappers/unity), [Matlab](./wrappers/matlab), [OpenNI](./wrappers/openni2), [UnrealEngine4](./wrappers/unrealengine4) and more to come. | |
+### 2️. Build from Source
+For a more custom installation, follow these steps to build the SDK from source.
+1. Clone the repository and create a build directory:
+   ```bash
+   git clone https://github.com/IntelRealSense/librealsense.git
+   mkdir build && cd build
+   ```
+2. Run CMake to configure the build:
+    ```bash
+    cmake .. -DBUILD_EXAMPLES=true
+    ```
+3. Build the project:
+    ```bash
+    make -j$(nproc)
+    ```
 
+### Setup Guides - build from source
+<div align="center" style="margin: 20px 0;">
+<a href="./doc/installation.md"><img src="https://img.shields.io/badge/Ubuntu_Guide-333?style=flat&logo=ubuntu&logoColor=white" style="margin: 5px;" alt="Linux Guide"/></a>
+<a href="./doc/installation_jetson.md"><img src="https://img.shields.io/badge/Jetson_Guide-333?style=flat&logo=nvidia&logoColor=white" style="margin: 5px;" alt="Jetson Guide"/></a>
+<a href="./doc/installation_windows.md"><img src="https://custom-icon-badges.demolab.com/badge/Windows_Guide-333?logo=windows11&logoColor=white" style="margin: 5px;" alt="Windows Guide"/></a>
+<a href="./doc/installation_osx.md"><img src="https://img.shields.io/badge/macOS_Guide-333?style=flat&logo=apple&logoColor=white" style="margin: 5px;" alt="macOS Guide"/></a>
+
+</div>
+
+## Python Packages
+[![pyrealsense2](https://img.shields.io/pypi/v/pyrealsense2.svg?label=pyrealsense2&logo=pypi)](https://pypi.org/project/pyrealsense2/)
+[![PyPI - pyrealsense2-beta](https://img.shields.io/pypi/v/pyrealsense2-beta.svg?label=pyrealsense2-beta&logo=pypi)](https://pypi.org/project/pyrealsense2-beta/)
+
+**Which should I use?**
+- **Stable:** `pyrealsense2` — validated releases aligned with SDK tags (Recommended for most users).  
+- **Beta:** `pyrealsense2-beta` — fresher builds for early access and testing. Expect faster updates.  
+> Note: until we provide a beta `apt install`, using the latest beta may require **cloning the development branch and compiling from source**.
```

Comment:
This note is not related to python.
It should be added above under distributions
And say that for minor releases we add Debians to the release artifacts that users can download install